libs/barista-components/table/src/table.ts

```diff
@@ -97,6 +98,7 @@ export class DtTable<T> extends _DtTableBase<T> implements OnDestroy {
   private _loading: boolean;
   private _destroy$ = new Subject<void>();
   private _showExportButton: boolean = false; //Revert to opt-in instead of opt-out per request
+  private _exportBlackList: string[] = [];
```

Comment:
We are trying to use inclusive language here, please rename this
```suggestion
  private _exportBlockList: string[] = [];
```